includes/class-scheduler.php

```diff
@@ -75,6 +78,8 @@ function ( $comment_id ) {
 
 		// profile updates for user options
 		if ( ! is_user_type_disabled( 'user' ) ) {
+			\add_action( 'delete_user', array( self::class, 'schedule_profile_delete' ), 10, 3 );
+			\add_action( 'deleted_user', array( self::class, 'schedule_user_delete' ), 10, 3 );
```

Comment:
As I mentioned in the comments: what about capability changes?

Reply:
@pfefferle since capability changes don't have any associated action hooks, we might consider adding some sort of user management section to the plugin settings.

Reply:
We could add some actions here https://github.com/Automattic/wordpress-activitypub/blob/f6b4783dedbc5ad07c42a9c49a275ecaed05a227/includes/class-admin.php#L450

Reply:
@pfefferle Should we tackle this in a followup PR?